content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -102,18 +102,16 @@ By default, a waypoint will only handle traffic destined for **services** in its
 
 It is also possible for the waypoint to handle all traffic, only handle traffic sent directly to **workloads** (pods or VMs) in the cluster, or no traffic at all. The types of traffic that will be redirected to the waypoint are determined by the `istio.io/waypoint-for` label on the `Gateway` object.
 
-The `--for` parameter to `istioctl waypoint apply` can be used to change the [traffic type](#waypoint-traffic-types) redirected to the waypoint:
+Use the `--for` flag on `istioctl waypoint apply` to change the types of traffic that can be redirected to the waypoint:
```

Comment:
```suggestion
Use the `--for` argument to `istioctl waypoint apply` to change the types of traffic that can be redirected to the waypoint:
```

Nit, this is not a flag (a binary choice based on the presence of the parameter). That said, our reference docs use the term 'flag' here.